helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -586,6 +586,26 @@ processors:
         new_name: container.memory.usage
   {{- end }}
 
+  {{- if or .Values.autodetect.prometheus .Values.autodetect.istio }}
+  # This processor is used to remove excessive istio attributes to avoid running into the dimensions limit.
+  # This configuration assumes single cluster istio deployment. If you run istio in multi-cluster scenarios,
+  # you may need to adjust this configuration.
+  attributes/istio:
+    include:
+      match_type: regexp
+      metric_names:
+        - istio_.*
+    actions:
+      - action: delete
+        key: source_cluster
```

Comment:
would it make sense to add a new value and generate each action so it's more flexible as additional attributes are (un)wanted?

Reply:
Do you mean having another option in values.yaml for that?

I wouldn't add more configuration options to the already bloated values.yaml for this niche use case.

Overriding the set of attributes is already possible, just a bit more verbose, like:
```yaml
agent:
  config:
    processors:
      attributes/istio:
        actions:
          - action: delete
            key: another_istio_attribute
          ...
```

Reply:
Also, the current interface to enable istio metrics is pretty unflexible.

```yaml
autodetect:
  istio: true
```

We can probably rethink it to be able to add this list of attributes. Maybe as a follow-up effort. Let me know if you have any ideas

Reply:
The problem is the transition from bool value to the map in a backward-compatible way. We can try something out but I would rather do it as a follow-up effort not blocking the release

Reply:
Filed this https://github.com/signalfx/splunk-otel-collector-chart/issues/769